### ISSUE TITLE
Parameters setting

### ISSUE THREAD

**Issue by Tokyo-dream** [3dlg-hcvc/paris#8]
This is the result trained through pretrain/parsed.yaml
![0](https://github.com/3dlg-hcvc/paris/assets/63401220/e987abb4-d676-4f64-a491-d3d2c869d0d5)
This is the result trained by pretrained model
![0](https://github.com/3dlg-hcvc/paris/assets/63401220/ea2e0228-0e54-42ed-baac-e1c7e6209602)
How can I set the parameters to make the mobile part more accurate?

**Comment by Tokyo-dream** [comment]
Thanks for this amazing work !

**Comment by SevenLJY** [comment]
Hi @Tokyo-dream. Thanks for your interest in our work!

As you’ve noted, the training for each object is essentially an optimization process, which can be easily affected by the randomness. I was also encountering the unstable training issue you mentioned in my practice. To get better results, it sometimes helps to tweak the learning rate a little bit or just train multiple times.

From my experience, there are varying degrees of training difficulty across these ten cases. Objects with relatively thin parts present a particular challenge as there are fewer opportunities for these regions to establish the correct correspondence. This can be considered a limitation inherent to our method. Increasing the number of views can, in many cases, help to enhance the likelihood of obtaining consistent positive results.
